### DISCUSSION TITLE
I NEED MOAR EMOTES.

### DISCUSSION THREAD

**Discussion by dwreyyes** [CubicRP/Cubic#15] [Ideas]
As I try to improve the scope of my roleplay, as well as try to improve the immersion and experience behind Weazel Media, I have to be honest and say that our emotes are nothing you can't find anywhere else. I'm not suggesting we could spend tons of money on emote packs, not really sure how they are monitored compared to maps and interiors, but I don't think adding new emotes would hurt anyone. 

I think that this will improve overall QoL within the server, and it would also broaden the horizon for roleplay, clips, and of course the laughs. Below I have provided a few links with animations. These are all QBcore scripts.

https://forum.cfx.re/t/paid-addon-10-roleplay-animations-and-gestures-by-mikey/4805676 (PAID)

https://tritons-store.tebex.io/category/animations - this guy coincidentally enough has the same script as the above link posted as well LMAO.

Turns out, there is much out there in terms of animation/emote packs, but I hope it's something we can keep our eye on!

**Comment by Gibuso** [comment]
MOAR EMOTES! MOAR EMOTES!

**Comment by cheeseburgerellie** [comment]
I saw this today while looking for the 3800 No Pixel Dances..maybe something to consider?

https://forum.cfx.re/t/trundles-roleplay-animations-updated-7-11-19/171735
